django/contrib/admin/templates/admin/delete_confirmation.html

```diff
@@ -20,21 +20,21 @@
 
 {% block content %}
 {% if perms_lacking %}
-    <p>{% blocktrans with escaped_object=object %}Deleting the {{ object_name }} '{{ escaped_object }}' would result in deleting related objects, but your account doesn't have permission to delete the following types of objects:{% endblocktrans %}</p>
+    <p>{% blocktrans with escaped_object=object force_escape %}Deleting the {{ object_name }} '{{ escaped_object }}' would result in deleting related objects, but your account doesn't have permission to delete the following types of objects:{% endblocktrans %}</p>
```

Comment:
I think everywhere, where it's not intended to have HTML tags in the translatable string inside other HTML tag, https://www.w3.org/TR/html4/charset.html#h-5.3.2, but I would also like to know @claudep opinion.